app/javascript/src/Policies/styles/policies.scss

```diff
@@ -8,6 +8,7 @@ $goColor: #4a90e2;
 $doColor: #54b45d;
 $dontColor: #ca4a4b;
 $subtleColor: #ddd;
+$error-color: #c00;
```

Comment:
Shouldn't this vars be moved to _colors.scss ?

Reply:
same as below